public/js/app/templates/bookmarkletTemplate.handlebars

```diff
@@ -101,6 +105,22 @@
   <div class="row">
     <div class="col-md-12">
       <h1>Share on FreeFeed</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12 select2-container">
```

Comment:
And `line-height` should only change internal items in select2, not the whole row elements.